toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -0,0 +1,539 @@
+# Mariner Image Customizer configuration
+
+Top-level type: [Config](#config)
+
+Operation ordering:
+
+1. Override the `/etc/resolv.conf` file with the version from the host OS.
+
+2. Update packages:
+
+   1. Remove packages ([PackageListsRemove](#packagelistsremove-string),
```

Comment:
Do we want to add an explicit note on changing installed kernel?

Reply:
I added a section about replacing packages. And included the kernel package as an example.